mllib/src/main/scala/org/apache/spark/mllib/stat/Statistics.scala

```diff
@@ -81,10 +85,14 @@ object Statistics {
    * @param x RDD[Double] of the same cardinality as y.
    * @param y RDD[Double] of the same cardinality as x.
    * @return A Double containing the Pearson correlation between the two input RDD[Double]s
+   * @since 1.1.0
    */
   def corr(x: RDD[Double], y: RDD[Double]): Double = Correlations.corr(x, y)
 
-  /** Java-friendly version of [[corr()]] */
+  /**
+   * Java-friendly version of [[corr()]]
+   * @since 1.4.1
+   */
```

Comment:
Actually, I see this in the API docs from 1.4.0
http://spark.apache.org/docs/1.4.0/api/scala/index.html#org.apache.spark.mllib.stat.Statistics$